README.md

```diff
@@ -88,39 +88,39 @@ The temporary directory that runners use to load jasmine files.
 Automatically deleted upon normal runs.
 
 #### options.outfile
-Type: `String`  
+Type: `String`
```

Comment:
the double spaces where there for markdown formatting. did your editor remove those or was that on purpose?

Reply:
This was purely my editor. I will remove these changes to the minimal set.